CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+
+### Pull Request
+
+When you're finished with the changes on your code, create a [pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests), also known as a PR.
+
+* You need to make a PR into the same branch that you created a sub branch from. Typically `main` or `develop`, unless you are proposing a modification to a feature branch or a bugfix branch.
+* Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one.
+
+### Code Review
+
+A pull request must be reviewed by **at least one other project member** to get merged quickly.
+
+> A CCDH engineer must review and accept your pull request. A code review (which happens with both the contributor and the reviewer present) is required for contributing.
+
+* You can add `unittest` tests to the repo as well.
+* We may ask for changes to be made before a PR can be merged, either using suggested changes or pull request comments. You can make any other changes and commit them in your branch.
+* As you update your PR and apply changes, mark each conversation as resolved.
+* If you run into any merge issues, checkout this [git tutorial](https://lab.github.com/githubtraining/managing-merge-conflicts) to help you resolve merge conflicts and other issues.
+
+## Development environment setup
+
+1. Install package dependencies.
+   1. Install poetry if necessary. One-time installation commands are available for [osx/linux/bash on windows](https://github.com/python-poetry/poetry#osx--linux--bashonwindows-install-instructions) and for [windows powershell](https://github.com/python-poetry/poetry#windows-powershell-install-instructions).
+2. Clone the ccdhmodel package repository.
+
+    ```shell
+    git clone https://github.com/cancerDHC/ccdhmodel.git
+    ```
+
+3. Create and activate the virtual environment.
+4. Run the following commands to build ccdhmodel and install the package along with all of its dependencies:
+
+    ```shell
+    cd ccdhmodel  # change directory to ccdhmodel
+    git checkout main  # switch to main branch of ccdhmodel
+    poetry build # build source and wheel archives
+    pip install dist/crdchmodel-x.y.z-py3-none-any.whl  # install wheel file
+    ```
+
+    **note**: x.y.z is the version number listed on pyproject.toml.
+
+5. To test new changes made to any of the modules within sheet2linkml, do the following:
+
+    ```shell
+    # make changes to any files or modules
+    pip uninstall ccdhmodel  # uninstall package
+    poetry build
+    pip install dist/crdchmodel-x.y.z-py3-none-any.whl  # install wheel file
+    ```
+
+* If you want to generate LinkML artifacts, follow the instruction on [Generation of LinkML artifacts](https://github.com/cancerDHC/ccdhmodel#generation-of-linkml-artifacts).
+
+## Format code with `black`
```

Comment:
I would suggest moving the `black` information up into the "Create a branch" section above, since that's where people are actually going to be doing development work :). Alternatively, it might be a good idea to move the entire "Development environment setup" section to the top of this document, followed by a section on how to submit issues/create branches/make PRs. What do you all think?

Reply:
Yes, moving the "development environment setup" section to the top of the document sounds good.